src/main/resources/reference.conf

```diff
@@ -0,0 +1,47 @@
+kamon {
+  cloudwatch {
+
+    # make sure AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY are set correctly in environment variables
+
+    # namespace is the AWS Metrics custom name-space
+    name-space = kamon-cloudwatch
+    # AWS region
+    region = eu-west-1
```

Comment:
this is probably also a job for the getCurrentRegion API call, so we'd only use this setting if not on EC2